### PR TITLE
Add Core SDK export for Node client

### PR DESCRIPTION
**Pull request by kevinwhereby**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -108,11 +115,12 @@
...
         "@reduxjs/toolkit": "^2.0.1",
         "@swc/helpers": "^0.3.13",
-        "@whereby/jslib-media": "whereby/jslib-media.git#1.4.1",
+        "@whereby/jslib-media": "file:.yalc/@whereby/jslib-media",
         "axios": "^1.2.3",
         "btoa": "^1.2.1",
         "events": "^3.3.0",
-        "heresy": "^1.0.4"
+        "heresy": "^1.0.4",
+        "ws": "^8.15.1"
```
Comment (nandito) [inline package.json:123]: Where do we use `ws`? 🤔

File: rollup.config.js [hunk 2]
```diff
@@ -31,10 +31,10 @@ const replaceValues = {
         "process.env.AP_ROOM_BASE_URL": JSON.stringify(process.env.AP_ROOM_BASE_URL),
         "process.env.RTCSTATS_URL": JSON.stringify(process.env.RTCSTATS_URL || "wss://rtcstats.srv.whereby.com"),
         "process.env.REACT_APP_API_BASE_URL": JSON.stringify(
-            process.env.REACT_APP_API_BASE_URL || "https://api.whereby.dev"
+            process.env.REACT_APP_API_BASE_URL || "https://embedded-ip-127-0-0-1.hereby.dev:4090"
```
Comment (nandito) [inline rollup.config.js:34]: why don't you just use the env var instead?

File: package.json (hunk 1 above)
Comment (kevinwhereby) [reply package.json:123]: oops

File: rollup.config.js (hunk 2 above)
Comment (kevinwhereby) [reply rollup.config.js:34]: I think I did this after attempting to use the env var for a few seconds and failing

Pure laziness 😅

File: package.json [hunk 3]
```diff
@@ -109,7 +116,7 @@
     "dependencies": {
         "@reduxjs/toolkit": "^2.0.1",
         "@swc/helpers": "^0.3.13",
-        "@whereby/jslib-media": "whereby/jslib-media.git#1.4.1",
+        "@whereby/jslib-media": "whereby/jslib-media.git#kevinhanna/support-explicit-device-handler",
```
Comment (sebapawlak) [inline package.json:119]: Please update, after releasing jslibmedia

File: rollup.config.js [hunk 4]
```diff
@@ -1,3 +1,5 @@
+const dotenv = require("dotenv");
```
Comment (sebapawlak) [inline rollup.config.js:1]: Is that necessary? I don't see any code that relies on the dotenv later on...

**Review by sebapawlak — APPROVED** [review]

File: rollup.config.js (hunk 4 above)
Comment (kevinwhereby) [reply rollup.config.js:1]: Where we inject the signal / api server addresses on line 42+, right?

**Pull request merged**